README.md

```diff
@@ -63,14 +63,30 @@ yarn android
 
 The app should now open in a connected device or a running emulator and fetch the JavaScript code from the running packager.
 
-To compile and run the iOS variant of the app, use:
+To compile and run the iOS variant of the app using the _default_ simulator device, use:
 
 ```
 yarn ios
 ```
 
 which will attempt to open your app in the iOS Simulator if you're on a Mac and have it installed.
 
+### Running on Other iOS Devices
+
+To compile and run the app using a different device, open the target device in Xcode by going to `Hardware > Device`, then use:
```

Comment:
I believe that having the simulator running prior the `yarn` command is not strictly necessary. `yarn ios` will launch it if it's not running already.

Reply:
Ah ok, I was having issues with launching others straight-away and this worked, so I assumed this was the "proper" way to do it – but now launching straight-away is working so I'll remove the part specific to opening prior to running the command.